### ISSUE TITLE
google bot noindex problem

### ISSUE THREAD

**Issue by azamet90** [frostschutz/MyBB-Google-SEO#74]
my sitemaps have noindex tag and google bot is not able to crawl my site how can i fix this? noindex only on sitemap ... please help me thx

**Comment by frostschutz** [comment]
search robots should use the sitemap for themselves only, they should not index it to show end users in search results.

thus the sitemap is noindex. this works fine for google and others. it should not affect your site in any way.

**Comment by azamet90** [comment]
> search robots should use the sitemap for themselves only, they should not index it to show end users in search results.
> 
> thus the sitemap is noindex. this works fine for google and others. it should not affect your site in any way.

![image](https://user-images.githubusercontent.com/44042178/85925294-108a5200-b8a0-11ea-874c-4aae3585be0c.png)
really? what is this then :(

**Comment by frostschutz** [comment]
Sorry, I can't read that and no context is given. Is that referring to sitemap-*.xml specifically or some other page?

Are you trying to get the XML Sitemap itself indexed? That's unusual and not supported. Even so, just remove the X-Robot line from inc/plugins/google_seo/sitemap.php then.

If you have this tag appearing on other pages, it's nothing to do with my plugin, can't help you there.

**Comment by azamet90** [comment]
> Sorry, I can't read that and no context is given. Is that referring to sitemap-*.xml specifically or some other page?
> 
> Are you trying to get the XML Sitemap itself indexed? That's unusual and not supported. Even so, just remove the X-Robot line from inc/plugins/google_seo/sitemap.php then.
> 
> If you have this tag appearing on other pages, it's nothing to do with my plugin, can't help you there.

shortly it says: Is it allowed to be indexed? **NO**

**Comment by frostschutz** [comment]
This was cut off from your screenshot but I assume you're referring to this?

![Screenshot_2020-06-28 URL Inspection](https://user-images.githubusercontent.com/194658/85951570-31b47680-b964-11ea-8e6f-79c5b42123a7.jpg)

Again, this is normal. You are not supposed to URL inspect the sitemap-index.xml (or any other sitemap subpage). It's not supposed to be indexed at all.

You can add and inspect the sitemap under https://search.google.com/u/0/search-console/sitemaps or specify it in your robots.txt and it'll work.

**Comment by azamet90** [comment]
> You can add and inspect the sitemap under

yes im trying tto add there "forum/sitemap-index.xml" but i get this error :(

**Comment by azamet90** [comment]
problem still exist

**Comment by azamet90** [comment]
i can add u as user to my google search console panel and u can see that it is not working because of that noindex problem?

**Comment by roast247** [comment]
did you get it fixed?
